editor/types/nonCr3.ts

```diff
@@ -125,3 +127,9 @@ export type NonCr3ValidationError = {
   fieldName: string;
   message: string;
 };
+
+export type NonCR3Record = {
```

Comment:
I was going to suggest that you construct this type using `<Partial<NonCr3Upload>>`, but it does feel right to have a separate type for this post-import record.